values/jobs/keycloak.gotmpl

```diff
@@ -37,9 +40,10 @@ init:
       {{ if $hasStagingCerts }}NODE_EXTRA_CA_CERTS=/tmp/node/certificates.crt {{ end }}binzx/otomi wait-for https://keycloak.{{ $v.cluster.domainSuffix }}
 image:
   repository: otomi/tasks
-  tag: v0.2.10
-  # tag: master
-  # pullPolicy: Always
+  # tag: v0.2.10
+  # TODO:
+  tag: keycloak-idp
+  pullPolicy: Always
 name: keycloak
 env:
   KEYCLOAK_ADDRESS: https://keycloak.{{ $v.cluster.domainSuffix }}
```

Comment:
I recommend to have a clear indicator that keycloak is and IdP and then do not provide env with `null` values.